### ISSUE TITLE
Eloquent Models Namespacing

### ISSUE THREAD

**Issue by lancepioch** [hardwiregaming/hardwire.community#20]
The idea is that we can more easily keep things namespaced. Also I believe that Steam Games (maybe Groups too?) should be cached in the database on our end. Their steam game ids will never change and neither will steam group ids. Once every hour or day we can update the list from all members.

![](http://yuml.me/b08ad12f)
```
[note: Yellow = Directory]
[app{bg:yellow}]->[Models{bg:yellow}]
[Models]->[Steam{bg:yellow}]
[Models]->[Teamspeak{bg:yellow}]
[Models]->[User]
[Models]->[Role]
[Steam]->[Game]
[Steam]->[Group]
[Teamspeak]->[Client]
```
http://yuml.me

**Comment by arcdigital** [comment]
Yeah, this sounds good - but what other groups would we be accessing besides ours?

**Comment by lancepioch** [comment]
I feel like this should be in the Wiki rather than an issue, do you agree?

**Comment by arcdigital** [comment]
@lancepioch you're right, I think that makes more sense too.
